assets/js/player.js

```diff
@@ -612,6 +612,9 @@ window.addEventListener('keydown', e => {
         case '7':
         case '8':
         case '9':
+            // Ignore numpad numbers
+            if (code > 57) break;
```

Comment:
Instead of using the code, let's just parse `decoratedKey` to an integer and multiply by 10. Wouldn't that work for both number sets?

Reply:
yeah, that should work. Though, I'm unsure about extending that feature to the numpad. I happen to often press numpad keys by mistake when reaching stuff next to my keyboard, and I didn't like the video jumping to the end...

Reply:
I've felt that pain before. I'm fine with that, though one thing I looked into is what those numbers actually are but the things I tested with showed the same numbers for the numpad and the other number row. Maybe it's something to do with me using an external keyboard on a macbook pro and is different on windows/linux